app/views/layouts/_user_menu.html.haml

```diff
@@ -16,6 +16,11 @@
     = link_to(conference_program_tracks_path(@conference.short_title)) do
       %span.fa.fa-road
       My Tracks
+-if @conference && @conference.program.cfps.for_booths.try(:open?)
```

Comment:
So if the cfp for booths is closed, I cannot see the link to the submissions I made while the cfp for booths was open. Why is that?